alpha-features.md

```diff
@@ -0,0 +1,28 @@
+# Open Service Broker API Alpha Features
+
+## Table of Contents
+  - [Alpha Overview](#overview)
+  - [How alpha features are documented](#how-alpha-features-are-documented)
+  - [Current alpha features](#current-alpha-features)
+
+## Overview
+
+The features described in this document are released at an ALPHA level.  They
+are are available for use in order for the community to provide feedback and
+are not yet an official part of the spec.
+
+Alpha features may be changed or removed in response to feedback.
+
+## How Alpha features are documented
+
```

Comment:
@mattmcneeney 

> Taking the JSON schema/parameters change as an example; there are changes to a number of responses in the specification, and having the entire specification together makes it easier for me to understand.

It looks to me that based on #165 that only the catalog response is changed.  Other requests are changed to include a note that brokers should validate the parameters that they are sent.

One concern that people have raised is that if we have multiple alpha features that each affect multiple parts of the spec, it may be difficult to tell exactly which parts of the combined spec correspond to specific features.